opacclient/tests/src/main/java/de/geeksfactory/opacclient/apis/WebOpacAtTest.java

```diff
@@ -0,0 +1,45 @@
+package de.geeksfactory.opacclient.apis;
+
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Objects;
+
+import de.geeksfactory.opacclient.objects.Library;
+import de.geeksfactory.opacclient.searchfields.SearchQuery;
+
+public class WebOpacAtTest {
+
+    WebOpacAt getApi() {
+        WebOpacAt api = new WebOpacAt();
+        Library lib = new Library();
+        lib.setData(new JSONObject(Collections.singletonMap("baseurl", "http://www.stbinnsbruck.web-opac.at")));
+        api.init(lib);
+        return api;
+    }
+
+    @Test
+    public void testParseTotalResults() {
+        assert WebOpacAt.parseTotalResults("Treffer pro Seite  •  1 - 10 von 46 Treffern") == 46;
+        assert WebOpacAt.parseTotalResults("Hits per page  •  1 - 10 of 46") == 46;
+        assert WebOpacAt.parseTotalResults("Sayfa başına sonuç  •  46 sonuçtan 1 - 10") == 46;
+    }
+
+    @Test
+    public void testParseCopyReturn() throws Exception {
+        assert Objects.equals(WebOpacAt.parseCopyReturn("1 (voraussichtl. bis 31.07.2015)"), "31.07.2015");
+
+    }
+
+    @Test
+    public void testSearchFields() throws Exception {
+        getApi().getSearchFields();
```

Comment:
@johan12345, what do you think on these tests, especially this and the next one? Do they make sense or should we incorporate them into the general unit tests? I see why they are useful, maybe more useful than the general ones (becuase they are faster to run etc.), but I don't like introducing API-specific single-library-dependent tests for such a generic thing.